introduction_to_amazon_algorithms/linear_learner_mnist/linear_learner_mnist.ipynb

```diff
@@ -270,6 +288,61 @@
     "linear.fit({\"train\": s3_train_data})"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Training with Automatic Model Tuning ([HPO](https://docs.aws.amazon.com/sagemaker/latest/dg/automatic-model-tuning.html)) <a id='AMT'></a>\n",
+    "***\n",
+    "As mentioned above, unstead of manually configuring our hyper parameter values and training with SageMaker Training, we'll use Amazon SageMaker Automatic Model Tuning.\n",
```

Comment:
```suggestion
    "As mentioned above, instead of manually configuring our hyper parameter values and training with SageMaker Training, we'll use Amazon SageMaker Automatic Model Tuning.\n",
```

Reply:
Fixed.